app/src/main/java/org/mozilla/tv/firefox/utils/Settings.kt

```diff
@@ -51,10 +52,11 @@ class Settings private constructor(context: Context) {
 
     fun shouldShowTVOnboarding(localeManager: LocaleManager, context: Context): Boolean {
```

Comment:
Should we have a test for this? Perhaps as a follow-up.

Reply:
I started writing a test for this, but it involves mocking context several layers deep.  The function is mostly declarative, so I don't think it's worth the time investment as is.  I think it would be better to use the time to separate out the logic entirely, as discussed in https://github.com/mozilla-mobile/firefox-tv/pull/2545#discussion_r303185118.